### PR TITLE
Many Birth by Sleep formats documented

### PR DESCRIPTION
**Pull request by Truthkey**
This commit includes documentation for the following Birth by Sleep formats:
* ArenaData
* ITB
* ITC
* PRIZEBOXDATA
* AAC
* ABC
* BDC
* MSS
* PCAM
* PVD
* cmdShop, mdlShop & iceShop
* cmdCharge
* AbiPattern
* StickerEd
* BGM
* FRR
* SEI
* AreaInfo
* Worldpoint
* BDD
* BIC
* BSD
* NGD
* NMD
* SCD
* TXA
* SEB

It also includes fixes for some other formats.

### CODE REVIEW

File: docs/bbs/AbiPattern.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+AbiPattern stands for *Ability Pattern*.
+
+This file stores a list of abilities. It's use is unknown, but it seems to be related to command melding.
+
+This file is contained within `Menu/Camp.arc`
+
+
+## Header
+| Offset | Type  | Description
```
Comment (Topaz-Reality) [inline docs/bbs/AbiPattern.md:11]: Leave a line between headers and everything else.
Github is known to break formatting otherwise.

**Review by Topaz-Reality — CHANGES_REQUESTED** [review]
Apply the comment I left to all files. Everything else seems gucci.

File: docs/bbs/AbiPattern.md (hunk 1 above)
Comment (Truthkey) [reply docs/bbs/AbiPattern.md:12]: Okay, it should be fixed now!

**Review by Vladabdf — CHANGES_REQUESTED** [review]
For KH2, we've added a "File types" hyperlink on its page with various formats listed beneath that link to their appropriate markdown documents. If you can make that table for BbS, this PR is golden to be approved. Everything else looks good since Topaz pointed out the header spacings and you fixed that. :thumbsup:

**Review by Topaz-Reality — CHANGES_REQUESTED** [review]
I am ready to approve it, however, what Vlad said makes sense.
Do that and it's approved.

File: docs/bbs/file-type.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
+# [Kingdom Hearts II](index.md) - file types
```
Comment (Delta-473) [inline docs/bbs/file-type.md:1]: I think you forgot to change this to Birth By Sleep ;)

Comment (Truthkey) [reply docs/bbs/file-type.md]: Yeah, sorry. I fixed it!

**Review by Vladabdf — APPROVED** [review]
Looks good. Awesome job!

**Review by Topaz-Reality — APPROVED** [review]
Is it jSut me, or is this PR lookin' kinda
_**T H I C C ?**_

**Pull request merged**